Makefile

```diff
@@ -11,7 +11,7 @@ endif
 
 REPO = github.com/operator-framework/api
 BUILD_PATH = $(REPO)/cmd/operator-verify
-PKGS = $(shell go list ./... | grep -v /vendor/)
+PKGS = $(shell go list ./...)
```

Comment:
We don't want to exclude any vendor packages anymore?

Reply:
We removed the vendor directory from source control in #213. But I can revert this change just in case it affects local dev for whatever reason.